validator/project_validator.go

```diff
@@ -1378,9 +1379,10 @@ func validateTVDependsOnTV(source, target model.TVPair, tvToTaskUnit map[model.T
 }
 
 type dependencyRequirements struct {
-	lastDepNeedsSuccess bool
-	requireOnPatches    bool
-	requireOnNonPatches bool
+	lastDepNeedsSuccess   bool
+	requireOnPatches      bool
+	requireOnNonPatches   bool
+	requireOnGitTagBuilds bool
```

Comment:
This value is never set.

Reply:
Thanks for catching, this is old from before I realized this was unnecessary.